test/integration/acceptance/custom/hello_policy/token_test.go

```diff
@@ -16,6 +18,7 @@ import (
 // And check whether it works or is disallowed by policy
 func createTokenPolicyScenario(cluster *base.ClusterContext, prefix, testPath, name string, works bool) (createToken cli.TestScenario) {
 
+	_ = os.MkdirAll(testPath, 0755)
 	createToken = cli.TestScenario{
```

Comment:
This has actually been done on the main test, as the tmp directory is used for tokens on more than one test function, and also for debug dumps